src/test/resources/projects/basic/archetype.properties

```diff
@@ -14,3 +14,4 @@ includeErrorHandler=n
 includeDispatcherConfig=n
 includeCommerce=n
 commerceEndpoint=https://hostname.com/grapql
+datalayer=y
```

Comment:
shouldn't it be `datalayer=${datalayer}` as for the below projects?

Reply:
`datalayer=${datalayer}` uses the default value. For `basic` I intended to use have it on, regardless.

Reply:
ok, makes sense.